site/docs/v1/tech/apis/scim/SCIMUser.adoc

```diff
@@ -0,0 +1,159 @@
+---
+layout: doc
+title: SCIM User APIs
+description: APIs for provisioning SCIM Users in FusionAuth using a SCIM Client
+---
+
+== Overview
+
+This page contains all of the APIs for managing Users through SCIM User requests.
+
+* <<Create a User>>
+* <<Delete a User>>
+* <<Retrieve a User>>
+* <<Update a User>>
+
+== Create a User
+
+This API is used to create a new FusionAuth User from a SCIM request.
```

Comment:
Different Lambdas would be called to process the incoming and outgoing requests so the dev would be able to retrieve data specific to one schema or the other and choose where to store it in the FusionAuth User. For example, they may want to store a manager's id that could be included in the EnterpriseUser schema into a user.data property since FusionAuth doesn't have that concept. The User schema would not have that data per the the spec so it's lambda would not try to store it.

In practice, the two lambdas will probably be pretty much identical, but this allows for the most flexibility if they want to support both the EnterpriseUser and User SCIM core schemas.